otter_welcome_buddy/common/utils/image.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from discord import Member
+from discord import User
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+from PIL.Image import Image as ImageType
+from PIL.ImageDraw import ImageDraw as ImageDrawType
+from PIL.ImageFont import FreeTypeFont
+
+from otter_welcome_buddy.common.constants import FONT_PATH
+
+_DEFAULT_IMAGE_PATH: str = os.path.dirname(os.path.realpath(__file__)) + "/image.jpg"
+_COLOR_TEXT: str = "black"
+_COLOR_OUTLINE: str = "gray"
+_COLOR_BACKGROUND: str = "white"
+
+
+def _get_size(txt: str, font: FreeTypeFont) -> tuple[int, int]:
+    """
+    Get the size (height, width) of the text provided based on the font in píxels
+    """
+    temp_img: ImageType = Image.new("RGB", (1, 1))
+    temp_draw: ImageDrawType = ImageDraw.Draw(temp_img)
+    _, _, text_width, text_height = temp_draw.textbbox((0, 0), txt, font)
+    return text_width, text_height
+
+
+def create_match_image(
+    week_otter_pairs: list[tuple[User | Member, User | Member]],
+) -> tuple[ImageType, str]:
+    """
+    Create a jpg image with the pairs of participants
+    """
+    first_list, second_list = zip(*week_otter_pairs)
+    first_column: str = "\n".join(
+        list(map(lambda user: f"{user.display_name}#{user.discriminator}", first_list)),
+    )
+    second_column: str = "\n".join(
+        list(
+            map(lambda user: f"{user.display_name}#{user.discriminator}", second_list),
+        ),
+    )
+
+    # TODO: except image creation errors
+    fontsize: int = 16
```

Comment:
this font size could also live in the conf file

Reply:
Same comment than above, but I can add this to the above vars.